### DISCUSSION TITLE
[@types/amplitude-js] Init config option `onExit` should be `onExitPage`

### DISCUSSION THREAD

**Discussion by Jackman3005** [DefinitelyTyped/DefinitelyTyped#59987] [Issues with a @types package]
In the amplitude init function there is a configuration field called `onExit` per [the documentation of the library](https://amplitude.github.io/Amplitude-JavaScript/Options/), it is actually `onExitPage`. I have changed my implementation to `onExitPage` and it works (whereas `onExit` does not work).

[a comment by an automated account is not shown]

**Reply by danmana** [reply]
Fixed in PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60037
